### PR TITLE
Bug 1826933: Show helm chart icon in group label in topology

### PR DESCRIPTION
**Pull request by rohitkrai03**
Fixes - https://issues.redhat.com/browse/ODC-3154

Screencast - 
Helmicons

### CODE REVIEW

**Comment by rohitkrai03** [comment]
cc: @openshift/team-ux-review

File: frontend/packages/console-shared/src/utils/icon-utils.ts [hunk 1]
```diff
@@ -6,9 +6,19 @@ export const getImageForCSVIcon = (icon: CSVIcon | undefined) => {
...
+  try {
+    // eslint-disable-next-line no-new
+    new URL(url);
+    return true;
+  } catch {
+    return false;
+  }
+};
+
+export const isIconBase64Encoded = (iconData: string): boolean => {
```
Comment (christianvogt) [inline frontend/packages/console-shared/src/utils/icon-utils.ts:19]: Is this really needed anymore?
`isIconUrl` will accept data encoded values.

**Pull request merged**